x-pack/plugins/apm/public/plugin.ts

```diff
@@ -181,16 +185,17 @@ const apmTutorialTitle = i18n.translate(
 );
 
 export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
+  private telemetry: TelemetryService;
   constructor(
     private readonly initializerContext: PluginInitializerContext<ConfigSchema>
   ) {
     this.initializerContext = initializerContext;
+    this.telemetry = new TelemetryService();
```

Comment:
Is it necessary with a stateful service? Could we just encapsulate this in a hook?